komga-webui/src/locales/en.json

```diff
@@ -519,33 +519,33 @@
     }
   },
   "error_codes": {
-    "ERR_1000": "File could not be accessed during analysis",
-    "ERR_1001": "Media type is not supported",
+    "ERR_1000": "Could access file during analysis",
+    "ERR_1001": "Unsupported media type",
     "ERR_1002": "Encrypted RAR archives are not supported",
     "ERR_1003": "Solid RAR archives are not supported",
     "ERR_1004": "Multi-Volume RAR archives are not supported",
-    "ERR_1005": "Unknown error while analyzing book",
-    "ERR_1006": "Book does not contain any page",
```

Comment:
Marked noun error

Reply:
The original problem was that it should have been
"The book does not contain any pages"
http://www.butte.edu/departments/cas/tipsheets/grammar/articles.html and page should be plural, since there are more pages.